docs/setup/quick-start.md

```diff
@@ -1,6 +1,6 @@
-There are some ways to setup Sedna:
-- If you have already an existing Kubernetes environment, you can follow [this instruction](./install.md)
-- If you have none Kubernetes environment, you can follow [the all-in-one instruction](./all-in-one.md)
-- If you want to develop Sedna, you can try [the local-up instruction](./local-up.md)
+There are some ways to set up Sedna, depends on your use case:
+- If you have none Kubernetes environment and don't want to install Kubernetes manually, you can follow [the instruction](./all-in-one.md) to install all-in-one Sedna environment.
+- Else you can follow [this instruction](./install.md) to install Sedna on existing Kubernetes cluster.
+- Also [there is a local up script](./local-up.md) which is mainly used for developing Sedna, it boots a local Kubernetes cluster and installs Sedna based on local Sedna repository.
```

Comment:
This version looks much more smooth. In fact, the local-up version itself is very attractive. We can polish the sentences a little bit to make it stand out. 
1) Highlight the difference between local up and others. Add examples to show why a local version is needed. 
2) The first half is already a completed sentence where a period should be used. 
3) "used for developing Sedna" could be misleading. It implies "one cannot develop Sedna based on all-in-one or normal-install".

Might be the following is clearer (not very sure about the sayings and pls correct me if anything wrong):

"One more thing: we also have a [local-up install script](./local-up.md) available for easier Sedna development, using only one machine to simulate multi-container environment. That is, instead of setting up several machines, the local-up version boots a local Kubernetes cluster and installs Sedna based on a local repository. 

For example, when one is contributing new features for Sedna before final deployment, codes like AI algorithms under testing can be frequently changed in the system. In that case, when coding, s/he would suffer from tortured re-installations and frequent failures of the whole complicated system. To avoid that, one can use [the local-up install](./local-up.md) for agile development and testing. After testing done and codes fixed, one can use [this instruction](./install.md) or [the all-in-one instruction](./all-in-one.md) for final installation and deployment."

Reply:
Add the example use case for local-up into local-up.md